services/QuillLMS/engines/comprehension/app/models/comprehension/activity.rb

```diff
@@ -51,5 +55,13 @@ def serializable_hash(options = nil)
     private def expire_turking_rounds
       turking_rounds.each(&:expire!)
     end
+
+    def log_creation(user_id)
+      log_change(user_id, :create_activity, self, nil, nil, nil, "Comprehension Activity #{id} - active")
```

Comment:
I'm not sure that there's a cleaner way to do this, but the use of `:create_activity` here is risky.  While Ruby symbols may feel more "static" than a simple string, they're just as prone to typos.  It'd be better if we could somehow make this a constant.  (That might mean making the individual values in the `ChangeLog` model into constants...)
```ruby
COMPREHENSION_ACTIONS = [
  COMPREHENSION_CREATE_ACTIVITY = 'Comprehension Activity - created',
  ...
]
```
You can reference something like this via `ChangeLog::COMPREHENSION_CREATE_ACTIVITY`.